schemas/test/en/test_progress_value_source_calculated_summary_extended.json

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
I think this issue was mainly down to poor schema design. I had added this schema just to test that the section dependencies were being captured in order when there were multiple sections (see the tests added in `test_questionnaire_schema`) and hadn't really thought about how it would work functionally.  I have updated it now so that it should work and also updated the section titles so that it is clearer on what is actually being tested here 😅

Reply:
Resolving to move discussion of this schema to the comment below 👍